### ISSUE TITLE
heygem-asr cpu占用率一直处于100%

### ISSUE THREAD

**Issue by zhanghuigreat** [GuijiAI/HeyGem.ai#148]
2025-03-11 20:00:08 
2025-03-11 20:00:08 [W shape_analysis.cpp:841] failed PropagateTensorShapeOnNode with schema: 
2025-03-11 20:00:08 aten::pad(Tensor self, int[] pad, str mode="constant", float? value=None) -> (Tensor)
2025-03-11 20:00:08 %input.967 : Float(*, *, *, requires_grad=0, device=cuda:0) = aten::pad(%input.965, %5646, %5647, %5648), scope: __module.model.encoders/__module.model.encoders.47/__module.model.encoders.47.self_attn/__module.model.encoders.47.self_attn.pad_fn # /usr/local/lib/python3.8/dist-packages/torch/nn/modules/padding.py:23:0
2025-03-11 20:00:08 
2025-03-11 20:00:08 [W shape_analysis.cpp:841] failed PropagateTensorShapeOnNode with schema: 
2025-03-11 20:00:08 aten::pad(Tensor self, int[] pad, str mode="constant", float? value=None) -> (Tensor)
2025-03-11 20:00:08 %input.987 : Float(*, *, *, requires_grad=0, device=cuda:0) = aten::pad(%input.985, %5760, %5761, %5762), scope: __module.model.encoders/__module.model.encoders.48/__module.model.encoders.48.self_attn/__module.model.encoders.48.self_attn.pad_fn # /usr/local/lib/python3.8/dist-packages/torch/nn/modules/padding.py:23:0

**Comment by zhanghuigreat** [comment]
更新代码后解决
